engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Number.java

```diff
@@ -122,6 +122,41 @@ private void registerInt64Methods(Language language, ModuleScope scope) {
         "ceil",
         org.enso.interpreter.node.expression.builtin.number.smallInteger.CeilMethodGen.makeFunction(
             language));
+    scope.registerMethod(
```

Comment:
All of these must also be implemented for big integers, otherwise we leak the distinction to the user.